.github/workflows/enso4igv.yml

```diff
@@ -30,7 +27,7 @@ jobs:
       - name: Update project version
         working-directory: tools/enso4igv
         run: |
-          mvn versions:set -DnewVersion=`mvn -q -DforceStdout help:evaluate -Dexpression=project.version | cut -f1 -d -`.$GITHUB_RUN_NUMBER
+          mvn versions:set -DnewVersion=`mvn -q -DforceStdout help:evaluate -Dexpression=project.version | cut -f1 -d -`.`expr $GITHUB_RUN_NUMBER - 1666`
```

Comment:
Every build of `enso4igv.yml` github action produces extension with a unique version. The [major and minor numbers](https://github.com/enso-org/enso/pull/7861/files#r1333129595) are taken from a `pom.xml`. The _"micro"_ version is the `$GITHUB_RUN_NUMBER`.

To avoid the _"micro"_ version to _reach the sky_, feel free to bump minor version](https://github.com/enso-org/enso/pull/7861/files#r1333129595) and then decrease the `$GITHUB_RUN_NUMBER` by a constant (currently `1666`) to start counting from zero again.